examples/imagenet/input_pipeline.py

```diff
@@ -126,21 +122,21 @@ def _decode_and_center_crop(image_bytes, image_size):
 
   offset_height = ((image_height - padded_center_crop_size) + 1) // 2
   offset_width = ((image_width - padded_center_crop_size) + 1) // 2
-  crop_window = tf.stack([offset_height, offset_width,
-                          padded_center_crop_size, padded_center_crop_size])
-  image = tf.image.decode_and_crop_jpeg(image_bytes, crop_window, channels=3)
+  crop_window = tf.stack([offset_height, offset_width, padded_center_crop_size,
```

Comment:
nit but please don't change whitespace unless there's a reason -- it makes merge conflicts unnecessarily likely

Reply:
reverted the changes.